### PR TITLE
Adds spinach to the game

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
SUMMARY: Content: Adds spinach to raw_veggy.json and the fridge group in item_groups

#### Purpose of change
Adds spinach to raw_veggy.json and the fridge group in item_groups. Can be eaten and used to add to recipes later. 

#### Describe the solution
Copied the values from cabbage and adjusted them.  Adjusted calories and  vitamins based on comparing spinach to cabbage on wikipedia. Adjusted spoils_in to roughly match the info on wikipedia. It rarely spawns in the fridge item group, 40% less then cabbage.

#### Describe alternatives you've considered
None

#### Additional context
None

image

### CODE REVIEW

**Comment by Rail-Runner** [comment]
Could you add spinach seeds as well, to grow it? Seems like it has enough resistance to cold to be grown in New England.

**Comment by snipercup** [comment]
> 
> 
> Could you add spinach seeds as well, to grow it? Seems like it has enough resistance to cold to be grown in New England.

Good idea! since this my first commit I try to keep it simple but will be sure to add it to the next pr.

File: data/json/items/comestibles/raw_veggy.json [hunk 1]
```diff
@@ -98,6 +98,29 @@
     "smoking_result": "dry_veggy",
     "vitamins": [ [ "vitC", 56 ], [ "calcium", 5 ], [ "iron", 5 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "spinach",
+    "name": "spinach",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/raw_veggy.json]: Please add a `name_plural` field.

Comment (snipercup) [reply data/json/items/comestibles/raw_veggy.json]: Added the field, thanks for reviewing

**Pull request merged**